contracts/SpokePool.sol

```diff
@@ -862,7 +827,7 @@ abstract contract SpokePool is
      * account. If depositor is a contract, then should implement EIP1271 to sign as a contract. See
      * _verifyUpdateV3DepositMessage() for more details about how this signature should be constructed.
      */
-    function speedUpV3Deposit(
+    function speedUpDeposit(
```

Comment:
Should we rename for consistency?
`UPDATE_V3_DEPOSIT_DETAILS_HASH` to `UPDATE_DEPOSIT_DETAILS_HASH`
https://github.com/across-protocol/contracts/blob/de61c0a2f06f4981a9352b0ed968acb7ae6337b9/contracts/SpokePool.sol#L140-L141

and
`UPDATE_V3_DEPOSIT_ADDRESS_OVERLOAD_DETAILS_HASH` to `UPDATE_V3_DEPOSIT_DETAILS_HASH`
https://github.com/across-protocol/contracts/blob/de61c0a2f06f4981a9352b0ed968acb7ae6337b9/contracts/SpokePool.sol#L145

Reply:
agreed, done.